www/_posts/2017-04-21-cordova-7.md

```diff
@@ -0,0 +1,140 @@
+---
+layout: post
+author:
+    name: Audrey So
+    url: https://twitter.com/aud_rey_so
+title:  "Apache Cordova 7.0.0"
+categories: news
+tags: news releases
+---
+
+We are happy to announce that `Apache Cordova 7.0.0` has been released!
+
+Most notable changes include:
+
+* [cordova@7.0.0] (https://www.npmjs.org/package/cordova)
+* [cordova-lib@7.0.0] (https://www.npmjs.org/package/cordova-lib)
+* [plugman@1.5.0] (https://www.npmjs.com/package/plugman)
+* [cordova-app-hello-world@3.12.0] (https://www.npmjs.com/package/cordova-app-hello-world)
+* [cordova-common@2.0.3] (https://www.npmjs.com/package/cordova-common)
+* [cordova-create@1.1.0] (https://www.npmjs.com/package/cordova-create)
+* [cordova-fetch@1.1.0] (https://www.npmjs.com/package/cordova-fetch)
```

Comment:
No space between the `[text]` and the `(href)`, otherwise it doesn't link properly :)